src/application/components/Explorer/Explorer.tsx

```diff
@@ -85,12 +85,13 @@ const iFrameStyles = css`
 `;
 
 const authorizeButtonStyles = css`
-  width: 175px;
+  width: 185px;
   height: 25px;
   cursor: pointer;
   border: none;
   border-radius: 2px;
   color: ${colors.grey.darker};
+  font-size: 13px;
```

Comment:
double checking these CSS changes are intentional? 👀

Reply:
yes! this was just something I saw when testing on firefox!